apps/microsoft-teams/frontend/src/hooks/useGetContentTypes.ts

```diff
@@ -18,17 +19,40 @@ const useGetContentTypes = () => {
   const getAllContentTypes = useCallback(async () => {
     try {
       setLoading(true);
-      // TODO: Implement pagination for content types
-      const contentTypesResponse = await sdk.cma.contentType.getMany({});
-      setAllContentTypes(contentTypesResponse.items || []);
+
+      const contentTypesSuperset: ContentTypeProps[] = [];
+      let skip = 0;
+      let needToFetchNextBatch = true;
+
+      while (needToFetchNextBatch) {
+        const contentTypesResponse = await sdk.cma.contentType.getMany({
+          query: {
+            limit: CONTENT_TYPE_LIMIT,
+            skip,
+          },
+        });
+
+        const totalNumOfContentTypes = contentTypesResponse.total;
+        contentTypesSuperset.push(...contentTypesResponse.items);
+
+        if (contentTypesSuperset.length >= totalNumOfContentTypes) {
+          // we have fetched all possible content types, exit while
+          needToFetchNextBatch = false;
+        } else {
+          // we need to fetch the next batch of content types, re-execute while
+          skip += CONTENT_TYPE_LIMIT;
+        }
+      }
+
+      setAllContentTypes(contentTypesSuperset);
     } catch (error) {
       const err = new Error('Unable to get content types');
       setError(err);
       console.error(error);
     }
 
     setLoading(false);
-  }, [sdk.cma.contentType]);
```

Comment:
Discussed with Lisa, we're not entirely sure why the linter did/didn't complain about `sdk.cma.contentType` or `sdk.cma.contentType.getMany`... but basically this was just to placate the linter.  There's 0 risk here, since the SDK is static, so I'll just leave this as is.

- [ ] leave it be.